### PR TITLE
Add slacker recipe for v0.9.16

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

**Comment by ericdill** [comment]
@os I am adding slacker to conda-forge for linux, mac and windows. This will allow users to easily install this with [conda] (http://conda.pydata.org/docs/) (conda install -c conda-forge slacker). The maintenance burden here is light as all testing and releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe which will be located at https://github.com/conda-forge/slacker-feedstock shortly after this PR is finished and merged. 

If you would like to be added as a maintainer (to make new releases for instance), I can happily add you to the maintainers list. If not, that is totally fine too!

**Comment by ericdill** [comment]
Failure on travis-ci looks unrelated to this recipe: `RuntimeError: HTTPError: 500 Server Error: Internal Server Error for url: https://binstar-cio-packages-prod.s3.amazonaws.com/ <snip>`

**Comment by os** [comment]
Sure. Thanks, @ericdill.

File: recipes/slacker/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        - slacker
+    requires:
+        - pytest
+        - mock
+    commands:
+        - py.test {{ environ.SRC_DIR }}/tests -vrsx
+
+about:
+    home: https://github.com/os/slacker
+    license: Apache
```
Comment (ocefpaf) [inline recipes/slacker/meta.yaml]: @ericdill please add the version here: https://github.com/os/slacker/blob/master/LICENSE#L2

Comment (ericdill) [inline recipes/slacker/meta.yaml]: Good catch. Updated the recipe.

File: recipes/slacker/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{% set version = "0.9.16" %}
+
+package:
+    name: tinydb
```
Comment (ocefpaf) [inline recipes/slacker/meta.yaml]: Argh! I guess that should be `slacker`, right?

**Pull request merged**